content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -147,23 +162,53 @@ password:
    You can transfer API credentials. Use this only when you have already created an account in
    another Odoo instance and wish to reuse it.
 
+A1 certificate upload
+~~~~~~~~~~~~~~~~~~~~~
+
+In order to issue electronic invoices a certificate needs to be uploaded to the `AvaTax portal
+<https://portal.avalarabrasil.com.br/Login>`_.
+
+The certificate will be synchronized with Odoo as far as in the AvaTax portal the external
+identifier number matches - without special characters - with the CNPJ number and in Odoo the
+identification number (CNPJ) matches with the CNPJ in AvaTax.
```

Comment:
```suggestion
The certificate will be synchronized with Odoo, as long as the external identifier number in the AvaTax portal matches - without special characters - with the CNPJ number, and the identification number (CNPJ) in Odoo matches with the CNPJ in AvaTax.
```
I believe I was able to get the gist of this, but if I am wrong feel free to change it.